### PR TITLE
Improve button styles

### PR DESCRIPTION
**Pull request by Aced14**
* Fix blue/cancel buttons that were showing up as generic ones
* Fix border colours on blue buttons
* Fix hover, active, focus and active+focus styles

**Screenshots...**

**Main page search...**
**Before + After:**
main-page-search-before

**Reference tooltips settings...**
**Before:**
reference-tooltips-config-popup-before
**After:**
reference-tooltips-config-popup-after

**Page preview settings...**
**Before:**
page-preview-config-overlay-before
**After:**
page-preview-config-overlay-after

**Edit page...**

**Normal...**
**Before:**
edit-page-before
**After:**
edit-page-after

**``:hover``**
**Before:**
edit-page-hover-before
**After:**
edit-page-hover-after

**``:active``**
**Before:**
edit-page-active-before
**After:**
edit-page-actve-after

**``:focus``**
**Before:**
edit-page-focus,active-focus-before
**After:**
edit-page-focus-after

**``:active:focus``**
**Before:**
edit-page-focus,active-focus-before
**After:**
edit-page-actve-after

### CODE REVIEW

**Comment by the-j0k3r** [comment]
Hello @Aced14! Thank you for reaching out with this contribution.

I like the changes proposed, as for the dark text within buttons and icons, all our styles sort have this styling in common, so It may look out of place.

Lets see what others say.. @Mottie @xt0rted @silverwind

**Comment by Mottie** [comment]
Our goal is to keep the dark style as close to the original as possible, but darker. So already blue buttons should be darker blue, bright white text inside those buttons should remain white, just not as bright. The Wikipedia style makes that incredibly hard in that most custom colors are added in-line with no specification to follow.

I don't think we really have hard-and-fast rules, so if you think something looks better one way or another, I say go for it... we can agree or disagree on a case-by-case basis... I'm okay with what you have in this PR so far, so that's an okay vote from me.

**Comment by Aced14** [comment]
Regarding the dark text in blue buttons... my line of thinking was that since Wikipedia's default font colour is really dark grey and is changed to light grey in this dark theme, it's more or less being inverted. By its very nature, the dark theme has to invert colours to some degree to change the site's overall colour scheme from bright to dark - but without going so far as to arbitrarily invert all colours.

In Wikipedia's normal theme, blue buttons seem to coloured as such to indicate a primary action. So to stand out compared to other buttons, they're coloured in bright blue and use an inverted font colour (white).

As far as this PR goes, I felt like using a black font for blue buttons would stay "truer" to what the normal Wikipedia theme was trying to achieve with its button colours (i.e. using an inverted colour to contrast with the default font colour).

**Comment by the-j0k3r** [comment]
My 2cents, Ide say staying true to original is important, but staying visible is more important, too dark  IMO.

**Comment by Aced14** [comment]
@the-j0k3r I'll change it back to white at some point tomorrow.

File: wikipedia-dark.user.css [hunk 1]
```diff
@@ -113,9 +113,32 @@ domain("wiki.rpcs3.net") {
...
   }
-  .mw-ui-button, .mw-ui-button *, button[type="submit"] {
+  .mw-ui-button, .mw-ui-button *, button[type="submit"],
+  .pure-button-primary-progressive,
+  .oo-ui-buttonElement-framed.oo-ui-widget-enabled.oo-ui-flaggedElement-primary.oo-ui-flaggedElement-progressive > .oo-ui-buttonElement-button,
+  .mw-ui-button.mw-ui-progressive {
     background-color: var(--base-color) !important;
-    color: #fff !important;
+    border-color: transparent !important;
+    color: #222 !important;
```
Comment (the-j0k3r) [inline wikipedia-dark.user.css]:
```suggestion
    color: #fff !important;
```

**Comment by Aced14** [comment]
Sorry, will take another day or two :(.

I've revised my commit (haven't pushed to GitHub yet), but want to rewrite its commit message, update the OP's screenshots and add comparison shots of hover/active/focus states. Ran out of time to do it tonight.

**Comment by the-j0k3r** [comment]
> I've revised my commit (haven't pushed to GitHub yet), but want to rewrite its commit message, update the OP's screenshots and add comparison shots of hover/active/focus states. Ran out of time to do it tonight.

No problem, if you want to do all of that, CSS change is all we need.

**Comment by Aced14** [comment]
Just forced-pushed an updated commit and reworked the OP.

The updated commit contains more changes than just the white text. So it might be worth re-reviewing in its entirety. I eliminated brightness filters since they were causing noticeable text blurring. Also tried moving things around to better fit within the different comment sections.

**Comment by the-j0k3r** [comment]
Thank you for your contribution. =)

**Pull request merged**